subcommands/keys/common.go

```diff
@@ -28,6 +29,59 @@ type TufSigner struct {
 	Key *rsa.PrivateKey
 }
 
+type keypair struct {
+	rsaPriv      *rsa.PrivateKey
+	atsPriv      client.AtsKey
+	atsPrivBytes []byte
+
+	atsPub      client.AtsKey
+	atsPubBytes []byte
+
+	keyid string
+}
+
+func GenKeyPair() keypair {
```

Comment:
i'm not sure it makes sense to make the function Public but the return type private. doesn't really matter in the big picture, it just grabbed my attention

Reply:
Done